test/csuite/wt2853_perf/main.c

```diff
@@ -193,6 +193,16 @@ main(int argc, char *argv[])
 		nfail += get_args[i].nfail;
 	}
 
+	/*
+	 * Note that slow machines can be skipped for this test.
+	 * See the bypass code earlier.
+	 */
+	if (nfail != 0)
+		fprintf(stderr,
+		    "ERROR: %d failures when a single commit"
+		    " took more than %d seconds.\n"
+		    "This may indicate a real problem or a"
+		    " particularly slow machine\n", nfail, MAX_GAP);
```

Comment:
You may want a full stop at the end of the second sentence.

Reply:
Thanks, fixed.